### PR TITLE
JDK-8307377: Remove use of `tagName` from TagletWriterImpl.linkSeeReferenceOutput

### PR DESCRIPTION
**Pull request by jonathan-gibbons**
Please review a simple fix to remove unnecessary detail from in-page and console messages about the name of the tag enclosing a bad reference.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307377](https://bugs.openjdk.org/browse/JDK-8307377): Remove use of `tagName` from TagletWriterImpl.linkSeeReferenceOutput

### Reviewers
 * [Hannes Wallnöfer](https://openjdk.org/census#hannesw) (@hns - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13811/head:pull/13811` \
`$ git checkout pull/13811`

Update a local copy of the PR: \
`$ git checkout pull/13811` \
`$ git pull https://git.openjdk.org/jdk.git pull/13811/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13811`

View PR using the GUI difftool: \
`$ git pr show -t 13811`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13811.diff">https://git.openjdk.org/jdk/pull/13811.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13811#issuecomment-1535199580)

### CODE REVIEW

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java [hunk 1]
```diff
@@ -479,7 +474,6 @@ private Content linkSeeReferenceOutput(Element holder,
                                            DocTree refTree,
                                            String refSignature,
                                            Element ref,
```
Comment (jonathan-gibbons) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java:476]: A related but different improvement would be to change this from `Element ref` to `TypeMirror refType`.

**Comment by jonathan-gibbons** [comment]
See also PR #

**Review by hns — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**